prdoc/pr_6353.prdoc

```diff
@@ -0,0 +1,12 @@
+title: Update litep2p network backend to version 0.8.0
+
+doc:
+  - audience: [ Node Dev, Node Operator ]
+    description: |
+      This release adds support for content provider advertisement and discovery to Kademlia protocol implementation (see libp2p
+      [spec](https://github.com/libp2p/specs/blob/master/kad-dht/README.md#content-provider-advertisement-and-discovery)).
```

Comment:
I don't think we should state this in the PRdoc, because this functionality is not exposed via substrate.